licensing.md

```diff
@@ -0,0 +1,27 @@
+# Licensing
+
+## Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S. English spelling.
+
+GitHub.com will still show licence details for the British English spelling.
+
+## Use MIT
+
+At GDS we use the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
I had a discussion with @willp-bl about this and he said "is it worth adding to that PR that MIT is generally the default but may not necessarily always be the one used?
e.g. some projects may have imported code from other open source code that has a pre-existing licence etc"

I think ^ that is a good point, but I don't think the suggested additional wording makes that clear and I'm not sure how to word it better. I'm be inclined to do as @fofr suggests and leave it as is for now until it needs updating because of a specific exception.